python-checks/src/test/java/org/sonar/python/checks/ClassMethodFirstArgumentNameCheckTest.java

```diff
@@ -19,13 +19,39 @@
  */
 package org.sonar.python.checks;
 
+import java.util.List;
 import org.junit.Test;
+import org.sonar.plugins.python.api.PythonCheck;
 import org.sonar.python.checks.utils.PythonCheckVerifier;
+import org.sonar.python.checks.utils.PythonQuickFixVerifier;
+import org.sonar.python.quickfix.IssueWithQuickFix;
+
+import static org.assertj.core.api.Assertions.assertThat;
 
 public class ClassMethodFirstArgumentNameCheckTest {
 
   @Test
   public void test() {
     PythonCheckVerifier.verify("src/test/resources/checks/classMethodFirstArgumentNameCheck.py", new ClassMethodFirstArgumentNameCheck());
+
+    String codeWithIssue = "class A():\n\n" +
+      "    @classmethod\n" +
+      "    def area(bob, length, width):\n" +
+      "        return length*width\n";
+    String codeFixed = "class A():\n\n" +
+      "    @classmethod\n" +
+      "    def area(cls, bob, length, width):\n" +
+      "        return length*width\n";
+
+    List<PythonCheck.PreciseIssue> issues = PythonQuickFixVerifier
+      .getIssuesWithQuickFix(codeWithIssue, new ClassMethodFirstArgumentNameCheck());
+
+    assertThat(issues).hasSize(1);
+    IssueWithQuickFix issue = (IssueWithQuickFix) issues.get(0);
+
+    assertThat(issue.getQuickFixes()).hasSize(1);
+
+    String codeQFApplied = PythonQuickFixVerifier.applyQuickFix(codeWithIssue, issue);
+    assertThat(codeQFApplied).isEqualTo(codeFixed);
```

Comment:
I would expect all of that to be handled in a single method call to PythonQuickFixVerifier, for example:
```
PythonQuickFixVerifier.verify(new ClassMethodFirstArgumentNameCheck(), codeWithIssue, codeFixed);
```